Assets/MRTK/Tests/PlayModeTests/ChangeActiveProfileTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.Boundary;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
```

Comment:
A test that I would like to see, going forward:

1. set the hololens 2 profile
2. Press a button, make sure it presses via poke
3. Set profile to be a hololens 1 profile
4. verify that poking does not press a button
5. Use head + airtap to press a button, verify that works.

Reply:
Definitely! We should have extensive tests for ensuring this works once all of the issues have been uncovered.

Reply:
Just to check @julenka you mean a test within this PR right?

Reply:
I will see if that scenario can work with just the changes in this PR. It may require additional changes to fully support the requested test.

Reply:
Looking at the profiles, there does not appear to be anything in the Default HoloLens1 profile that expressly disables near interaction.

I will update the profile as part of getting the above described test implemented.